modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -2315,6 +2315,12 @@ components:
                     format: date-time
                     readOnly: true
                     type: string
+                datePublished:
+                    # @review
+                    description:
+                        Date when the article will be published.
```

Comment:
Take a look at the description used in `datePublished` for other entities and try to follow a similar pattern.